src/components/quickstarts/QuickstartSort.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { navigate } from 'gatsby';
+import PropTypes from 'prop-types';
+
+const QuickstartSort = ({ className }) => {
+  const params = new URLSearchParams(location.search);
+  const sortParam = params.get('sort');
+  const handleChange = (event) => {
+    params.set('sort', event.target.value);
+    navigate(`/instant-observability?${params.toString()}`);
+  };
+  return (
+    <>
```

Comment:
Looks like we don't need this fragment!

Reply:
Looks like this comment got prematurely resolved.